curriculum/challenges/english/25-front-end-development/lecture-understanding-core-javascript-fundamentals/6732c69d82814160951b1aa7.md

```diff
@@ -70,7 +71,25 @@ console.log(undefinedNum); // NaN
 console.log(nullNum); // 0
 ```
 
-When working with arrays there are a few things to consider, an empty array will return `0`, an array with a single number will return that number, an array with multiple numbers returns `NaN`, and an array with strings will return `undefined`. 
+When working with arrays there are a few things to consider.
```

Comment:
here is the issue for this change 

https://github.com/freeCodeCamp/freeCodeCamp/issues/60219